apps/studio/prisma/seed.ts

```diff
@@ -14,6 +14,76 @@ import {
 } from '~/server/modules/resource/resource.types'
 import { db } from '../src/server/modules/database'
 
+const MOCK_PHONE_NUMBER = '123456789'
+
+const ISOMER_ADMINS = [
+  'alex',
+  'jan',
+  'kishore',
+  'jiachin',
+  'sehyun',
+  'harish',
+  'zhongjun',
+  'hanpu',
+]
+
+const PAGE_BLOB = {
```

Comment:
nit: this is quite tied to the schema so might want to consider having this in a central location so we know to update it when the schema is adjusted.

Reply:
i left it here because this is the initial `seed` script ran to populate your db with fake data. 

is there another place you were thinking of?